server/follow/FollowService.js

```diff
@@ -0,0 +1,77 @@
+'use strict'
+
+function follow (followService, meId, otherId) {
+  return Promise.all([
+    new Promise(function (resolve, reject) {
+      followService.redisClient.zadd(`following:${meId}`, Date.now(), otherId, function (err, result) {
+        if (err) return reject(err)
+        resolve(result)
+      })
+    }),
+    new Promise(function (resolve, reject) {
+      followService.redisClient.zadd(`followers:${otherId}`, Date.now(), meId, function (err, result) {
+        if (err) return reject(err)
+        resolve(result)
+      })
+    })
+  ])
```

Comment:
can you use async/await for these?

Reply:
I need to figure out how to rewrite those ugly lines...